photon-client/src/main/scala/com/linkedin/photon/ml/photon_io/README.md

```diff
@@ -0,0 +1 @@
+This package is slated for removal when photon disappears (and photon models are handled as Game fixed effect models).
```

Comment:
Yes - I am using "photon" to mean legacy here, as opposed to "game", but "photon" is still the product name. Maybe we should get rid of the name "game" when the legacy stuff is all removed.